changelog/feature.7935.md

```diff
@@ -0,0 +1 @@
+IBM Watson Training Data Importer. This will allow to train a rasa bot using IBM Watson V2 Training data downloadable from IBM Workspaces. 
```

Comment:
It allows to convert the training data and then train, but doesn't allow training _on_ the IBM watson training data, right?